core/src/main/java/io/aiven/klaw/service/KafkaConnectControllerService.java

```diff
@@ -79,6 +79,7 @@ public class KafkaConnectControllerService {
       OBJECT_MAPPER.writerWithDefaultPrettyPrinter();
   public static final TypeReference<ArrayList<ResourceHistory>> VALUE_TYPE_REF =
       new TypeReference<>() {};
+  public static final String PASSWORD_KEY = "password";
   @Autowired private CommonUtilsService commonUtilsService;
```

Comment:
Just wondering should we make this an @value(klaw.connectors.restrictedFields)
Have it as comma deliminated in the application.properties e.g.
klaw.connectors.restrictedFields=password,mySecretUrl,username
split by the comma into a string list
Then on line 569 you can check if the key is in the list and encrypt on that